client/Containers/Cart/Cart.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import ProductCard from './ProductCard';
+import Summary from './Summary';
+import styled from 'styled-components';
+
+const mapStateToProps = (state) => {
+  return {
+    cart: state.cart,
+  };
+};
+
+class Cart extends Component {
+  itemCount = () => {
+    let {
+      cart: { cartProducts },
+    } = this.props;
+    cartProducts = cartProducts || [];
+    if (!cartProducts.length) return 0;
+    return cartProducts.reduce((accum, item) => accum + item.quantity, 0);
```

Comment:
@vikkiwchan - I modified this reduce function to work for all array sizes, by giving it a default value of 0 as a second parameter, and changing the logic to simply add the current item quantity to the accumulator. The version before was breaking when the product quantity was greater than 2